minitests/litex/uart_ddr/arty/scripts/Makefile

```diff
@@ -0,0 +1,5 @@
+litex/litex/tools/litex_client.py:
+	git clone https://github.com/enjoy-digital/litex.git
```

Comment:
should we point to exact commit?

Reply:
I think it is not strictly necessary, but I have added a `git checkout` to the current commit, to be sure.